public/javascripts/questionnaires/questionnaire.js

```diff
@@ -1,40 +1,129 @@
-const input = document.querySelector('input');
+var currentTab = 0; // Current tab is set to be the first tab (0)
+showTab(currentTab); // Display the current tab
 
-// prevent enter button from submitting form
-input.onkeypress = function(e) {
-    if (!e) e = window.event;
-    var keyCode = e.code || e.key;
-    if (keyCode == 'Enter') {
-      e.preventDefault(); //prevent submitting form
+function showTab(n) {
+  // This function will display the specified tab of the form ...
+  var x = document.getElementsByClassName("tab");
+  x[n].style.display = "block";
+  // ... and fix the Previous/Next buttons:
+  if (n == 0) {
+    document.getElementById("prevBtn").style.display = "none";
+  } else {
+    document.getElementById("prevBtn").style.display = "inline";
+  }
+  if (n == (x.length - 1)) {
+    document.getElementById("nextBtn").innerHTML = "Submit";
+  } else {
+    document.getElementById("nextBtn").innerHTML = "Next »";
+  }
+  // ... and run a function that displays the correct step indicator:
+  fixStepIndicator(n)
+}
+
+function nextPrev(n) {
```

Comment:
where do you call this function? nextPrev(n)? I see you call on showTab

Reply:
this is the code we copied and didn't write btw :P but this function is called when you click on a button (see the ejs files where the buttons are)